Changed module name and make minor edits (#135)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_monitoring-resource-usage-of-openstack-services.adoc

// readers and search engines find information quickly.
// Do not start the title with a verb. See also _Wording of headings_
// in _The IBM Style Guide_.
[id="monitoring-resource-usage-of-openstack-services_{context}"]
= Monitoring the resource usage of {OpenStack} services

> Monitor the resource usage of the {OpenStack} services, such as the APIs and other infrastructure processes, to identify areas of the cloud that are under a lot of stress. Enable the `collectd-libpod-stats` plug-in to gather CPU and memory usage metrics for every container running in the overcloud.

Author: "to identify areas of the cloud that are under a lot of stress."

@pleimer Can you be more specific here? For example, to identify areas that need more compute power, something like that.

Reviewer: That probably is the best clarification:
"to identify areas that need more compute power"

Reviewer: If we want to include more details, we could add a phrase like: "and identify bottlenecks in the cloud"

Reviewer: "to identify bottlenecks in the overcloud by showing services running out of compute power" or something like that

> Monitor the resource usage of the {OpenStack} services, such as the APIs and other infrastructure processes, to identify areas of the cloud that are under a lot of stress. Enable the `collectd-libpod-stats` plug-in to gather CPU and memory usage metrics for every container running in the overcloud.

Author: ```suggestion
Monitor the resource usage of the {OpenStack} services, such as the APIs and other infrastructure processes, to identify areas of the cloud that require more compute power. Enable the `collectd-libpod-stats` plug-in to gather CPU and memory usage metrics for every container running in the overcloud.
```

> Monitor the resource usage of the {OpenStack} services, such as the APIs and other infrastructure processes, to identify areas of the cloud that require more compute power. Enable the `collectd-libpod-stats` plug-in to gather CPU and memory usage metrics for every container running in the overcloud.

Author: ```suggestion
Monitor the resource usage of the {OpenStack} services, such as the APIs and other infrastructure processes, to identify bottlenecks in the overcloud by showing services running out of compute power. Enable the `collectd-libpod-stats` plug-in to gather CPU and memory usage metrics for every container running in the overcloud.
```

.Prerequisites

* You have created the `stf-connectors.yaml` file. For more information, see xref:configuring-red-hat-openstack-platform-overcloud-for-stf_completing-the-stf-configuration[].
* You are using the most current version of {OpenStack}: {OpenStackVersion}.

.Procedure

> . Open the `stf-connectors.yaml`.

Author: ```suggestion
. Open the `stf-connectors.yaml` file.
```

. Add the following configuration to `parameter_defaults`:
+
```
  CollectdEnableLibpodstats: true
```

> . Continue with the overcloud deployment procedure.

Author: For future editing.
